interrogate_badge.svg

```diff
@@ -0,0 +1,58 @@
+<svg width="140" height="20" viewBox="0 0 140 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```

Comment:
I like the badge! However, it looks like this file is static, which means it won't update when the tests improve.

I suspect a dynamic badge tied to the develop branch can be implemented with the aid of GitHub artifacts (see: https://github.com/actions/upload-artifact). That may take a little while to figure out, though. Can I suggest that you remove the badge for now so that we can get this pull merged in. Then, if you're up for figuring out artifacts, see if you can set up another pull to add a dynamic badge?

Reply:
Sure, I have removed the badge, didn't notice it wasn't getting updated. 
I will look into the dynamic badge though.